InternetHospital.BusinessLogic/Services/FeedBackService.cs

```diff
@@ -0,0 +1,48 @@
+﻿using AutoMapper;
+using FluentValidation;
+using InternetHospital.BusinessLogic.FluentValidators;
+using InternetHospital.BusinessLogic.Interfaces;
+using InternetHospital.BusinessLogic.Models;
+using InternetHospital.DataAccess;
+using InternetHospital.DataAccess.Entities;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace InternetHospital.BusinessLogic.Services
+{
+    public class FeedBackService : IFeedBackService
+    {
+        private readonly ApplicationContext _context;
+
+        public FeedBackService(ApplicationContext context)
+        {
+            _context = context;
+        }
+
+        public FeedBack FeedBackCreate(FeedBackCreationModel model)
+        {
+            FeedbackValidator validations = new FeedbackValidator();
+            try
+            {
```

Comment:
remove try catch or add loging

Reply:
We used ValidateAndThrow() method and i think it is nice to use try,catch here

Reply:
in that case, add log message in catch block